package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "react-render-server",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "description": "A node.js server for server-side rendering Khan Academy react components.",
-  "main": "src/main.js",
+  "main": "dist/main.js",
   "scripts": {
-    "start": "ulimit -n 999999 || ulimit -n 2048; node src/main.js -p 8080 --log-level=debug",
-    "dev": "nodemon -i src/render_cache src/main.js -- --dev",
-    "debug": "nodemon -i src/render_cache src/main.js -- --dev --log-level=debug",
-    "profile": "node-debug -p 8040 --no-preload src/main.js --dev",
-    "serve_local": "nodemon -i src/render_cache src/main.js -- --dev",
-    "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js'",
+    "clean": "rm -rf dist",
+    "start": "ulimit -n 999999 || ulimit -n 2048; npm run build && node dist/main.js -p 8080 --log-level=debug",
+    "dev": "npm run build && nodemon dist/main.js -- --dev",
+    "debug": "npm run build && nodemon dist/main.js -- --dev --log-level=debug",
+    "profile": "npm run build && node-debug -p 8040 --no-preload src/main.js --dev",
+    "serve_local": "npm run build && nodemon dist/main.js -- --dev",
+    "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --compilers js:@babel/register",
     "deploy": "./deploy.sh",
     "deploy:docker": "DOCKER=1 ./deploy.sh",
     "set_default": "./set_default.sh",
-    "lint": "npx eslint --config ./eslint/eslintrc src",
-    "prettyquick": "npx pretty-quick"
+    "lint": "eslint --config ./eslint/eslintrc src",
+    "prettyquick": "pretty-quick",
+    "build": "npm run clean && babel src --out-dir dist --source-maps --ignore src/testdata/**/*.js,*_test.js",
+    "postinstall": "npm run build"
```

Comment:
This shouldn't be necessary, `npm build` is already called on `install`: https://docs.npmjs.com/cli/build

Reply:
Hmm, I was not having that experience, but perhaps it was because of a different error. I'll remove this and check all is well.

Reply:
Yeah, the build script definitely doesn't run on install for me without this.

Reply:
I suspect it's a difference between `npm install` or `yarn install` at the consumer of the package level, versus in the package directory.

With the postinstall step, it's running in the former case, which is what we want. Without it, it doesn.t

Reply:
Very strange - I'm able to replicate this locally, too! FWIW, `postinstall` is a synonym of `install`, as well.

Reply:
When you say replicate; you mean the behavior I see or the one you expect to see?

Reply:
@somewhatabstract The behavior you see - I downloaded your PR and replicated it locally; "build" is not automatically run by "install".

Reply:
OK, this probably goes away when we monorepo, so I think we can accept the weirdness for the interim.